src/shared/telemetry/telemetryLogger.ts

```diff
@@ -45,6 +45,10 @@ export class TelemetryLogger {
         return this._metrics.length
     }
 
+    public clear(): void {
+        this._metrics.length = 0
+    }
```

Comment:
TODO: TelemetryLogger feels somewhat unnecessary. TelemetryService already has a queue (which is flushed every few minutes), we can query that. If we want to query more logs (e.g. in tests), just increase the flush interval.